submissions/dashakim/tiny-world/index.js

```diff
@@ -0,0 +1,64 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+    constructor(species, name, gender, legs, hands, saying) {
```

Comment:
One of the major points of using classes in OOP is benefiting from class inheritance. This way you may define only what is really required for a class. I.e. animals won't have any hands rather than having zero hands.

Also defining classes like `Human`, `Dog` etc you may skip defining species at initialization.